src/utils/offer/OfferUtils.js

```diff
@@ -7,6 +7,7 @@ const HumanReadableErrors = Object.freeze({
     "must-be-admin": "Only admin users can do this action.",
     "login-required": "You must be logged in to do this action.",
     "must-be-company": "You must be a company to do this action.",
+    "max-concurrent-offers-reached:5": "You cannot have more than 5 concurrent visible offers.",
```

Comment:
The solution is not as simple as this, you shouldn't hard code the value `5`. 
You can take a look at how it's being done in [other places](https://github.com/NIAEFEUP/nijobs-fe/blob/d6d0d2fd0f08c6600857259112cc6dc11ea27067/src/components/Offers/New/CreateOfferUtils.js?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L51-L55).